### DISCUSSION TITLE
Multivariate flag weights is not assigned to identities correctly

### DISCUSSION THREAD

**Discussion by mohamedassem96** [Flagsmith/flagsmith#4402] [Q&A] [answered]
I'm trying to add four variants with 25% each and with having 4 identities, I'm trying to add four variants with 25% each and with having 4 identities

![image](https://github.com/user-attachments/assets/e2fb105f-8e92-4d79-af98-f4625d649376)
![image](https://github.com/user-attachments/assets/ada8cdc8-1631-4773-9e09-b8e5e5541261)
![image](https://github.com/user-attachments/assets/67c6624e-639c-4326-8fc1-75dc56d88a07)
![image](https://github.com/user-attachments/assets/24c53021-bbf9-4790-9ea0-658bc297bab0)
![image](https://github.com/user-attachments/assets/73177630-7283-4112-93ef-41f5175a6dad)
![image](https://github.com/user-attachments/assets/429cf990-3a74-4931-8447-697d5145636b)

**Comment by matthewelwell** [comment]
Hi @mohamedassem96, I don't think you've understood the difference between multivariate features and identity overrides correctly. These are 2 entirely separate concepts. Please read the documentation [here](https://docs.flagsmith.com/basic-features/managing-features#multi-variate-flags) on multivariate flags and [here](https://docs.flagsmith.com/basic-features/managing-identities#identity-overrides) on identity overrides. Thanks!

**Reply by mohamedassem96** [reply]
thanks for your response @matthewelwell 
I've not used  identity overrides yet, the values in screenshots is what's assigned by multivariate features to each user without applying any identity overrides

**Reply by matthewelwell** [reply, marked as answer]
Please read the documentation I sent. Multivariate flag values are assigned randomly based on the weightings provided. With 4 identities, you are not guaranteed to get 1 of each variation. At enough scale, the distribution will trend to 25% for each variation, but at lower scales you will likely see uneven distributions.
